source/features/raw-file-link.tsx

```diff
@@ -10,16 +10,22 @@ const createRawUrl = (href: string): string => {
 };
 
 function addRawButtons(): void {
-	for (const anchor of select.all<HTMLAnchorElement>('.js-file-header-dropdown [data-ga-click^="View file"]')) {
-		anchor.after(
-			<a href={createRawUrl(anchor.pathname)} className="pl-5 dropdown-item btn-link" role="menuitem">
-				View raw
-			</a>
-		);
+	for (const dropdown of select.all<HTMLElement>('.js-file-header-dropdown')) {
+		const anchor: HTMLAnchorElement | null = select('[data-ga-click^="View file"]', dropdown);
```

Comment:
In this case you can add a `!` at the end instead of specifying `|null`, and can specify the expected type with `<...>`

```
const anchor = select<HTMLAnchorElement>('[data-ga-click^="View file"]', dropdown)!;
```